systemd/crc-check-cloud-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if grep -qi 'ec2' /sys/class/dmi/id/product_uuid ||
+   grep -qi 'google' /sys/class/dmi/id/product_name ||
+   grep -qi 'microsoft' /sys/class/dmi/id/sys_vendor; then
```

Comment:
@anjannath can you also add `openstack` to the list using `grep -qi 'openstack' /sys/class/dmi/id/product_name`
```
$ cat /sys/class/dmi/id/product_name
OpenStack Compute
```

Reply:
Thanks, Updated!